src/vs/workbench/contrib/debug/browser/watchExpressionsView.ts

```diff
@@ -225,14 +228,28 @@ export class WatchExpressionsRenderer extends AbstractExpressionsRenderer {
 
 	static readonly ID = 'watchexpression';
 
+	constructor(
```

Comment:
Maybe add a dependency to `instantiationService` to the `AbstractExpressionsRenderer`. I think that would minimize changes